peripheral.xarcade/resources/resource.language.en_gb/strings.po

```diff
@@ -0,0 +1,25 @@
+# Kodi Media Center language file
+# Addon Name: X-Arcade Tankstick Driver
+# Addon id: peripheral.xarcade
+# Addon Provider: Team Kodi
+msgid ""
+msgstr ""
+"Project-Id-Version: KODI Addons\n"
+"Report-Msgid-Bugs-To: https://github.com/kodi-game/peripheral.xarcade/issues/\n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Kodi Translation Team\n"
+"Language-Team: English (United Kingdom) (http://www.transifex.com/projects/p/kodi-addons/language/en_GB/)\n"
```

Comment:
URL should be https://kodi.weblate.cloud/languages/en_gb/

Reply:
Has not thought about this line and taken from others. Then I have a good part on my next addon update round to change where is wrong :laughing: